account_banking_pain_base/models/account_payment_order.py

```diff
@@ -449,7 +449,12 @@ def generate_party_block(
             party_type_label = _("Creditor name")
         elif party_type == 'Dbtr':
             party_type_label = _("Debtor name")
-        name = 'partner_bank.partner_id.name'
+
+        if partner_bank.acc_holder_name:
+            name = 'partner_bank.acc_holder_name'
+        else:
+            name = 'partner_bank.partner_id.name'
+
```

Comment:
I think you can put an expression there:
```suggestion
        name = 'partner_bank.acc_holder_name or partner_bank.partner_id.name'
```
and please don't introduce empty lines inside a method.

Reply:
Yep, that works, here you go.